### DISCUSSION TITLE
Need Migrate-engine node modules for rhel

### DISCUSSION THREAD

**Discussion by mounikaberrybox** [prisma/prisma#20923] [Q&A]
### Question

Hi Team, 

Im using AWS RDS Postgres DB and using aws lambda functions to perform db migrations. To get the migration done, lambda environment node requires migrate-engine of prisma specific to RHEL OS. I have tried all the ways like have used AWS EC2 with RHEL OS to try to get the migrate-engine. But I'm not getting it. Please help me to make it work 

Thanks, 
Mounika Bomma

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS:  AWS RHEL
- Database:  POSTGRES
- Node.js version:  v16.20.1

- Run `prisma -v` to see your Prisma version and paste it

 prisma --version
prisma                  : 5.2.0
@prisma/client          : 5.2.0
Current platform        : rhel-openssl-3.0.x
Query Engine (Node-API) : libquery-engine 2804dc98259d2ea960602aca6b8e7fdc03c1758f (at ../../../prisma/node_modules/@prisma/engines/libquery_engine-rhel-openssl-3.0.x.so.node)
Schema Engine           : schema-engine-cli 2804dc98259d2ea960602aca6b8e7fdc03c1758f (at ../../../prisma/node_modules/@prisma/engines/schema-engine-rhel-openssl-3.0.x)
Schema Wasm             : @prisma/prisma-schema-wasm 5.2.0-25.2804dc98259d2ea960602aca6b8e7fdc03c1758f
Default Engines Hash    : 2804dc98259d2ea960602aca6b8e7fdc03c1758f
Studio                  : 0.494.0

**Comment by janpio** [comment]
I think `schema-engine-rhel-openssl-3.0.x` is what you are looking for. The `migration-engine` has been renamed to `schema-engine` a few releases ago.

**Reply by ludralph** [reply]
Hey @mounikaberrybox ,

do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
